### ISSUE TITLE
Add `create_quarto_project` or support Quarto-related parameters in `create_project`

### ISSUE THREAD

**Issue by elipousson** [r-lib/usethis#1891]
In exploring options for creating a R Project file, I found `create_project()` and wondered if it might be enhanced by the addition of new parameters to better support the creation of Quarto projects.

A simple `create_quarto_project()` that includes the basic `_quarto.yml` settings that is created when selecting "Quarto Project" from the New Project dialog in RStudio could be one option.

Another option could be a project type parameter, e.g. "project", "website", "book", "blog", corresponding to those same default settings.

One last (but not mutually exclusive) option could be updating `create_project()` to take a `config_file` parameter (similar to `use_pkgdown()`) and offering some way to define a set of files to use as a starting place (maybe using a `repo_spec` parameter similar to `use_standalone()`).

If there is interest in this feature, I'd be happy to work on it and open a pull request.

**Comment by jennybc** [comment]
I wonder if there are plans for anything like this in [quarto-r](https://github.com/quarto-dev/quarto-r) @jjallaire @cderv.

Where is the obvious home for it? 🤔

**Comment by cderv** [comment]
Yes definitely something we should do in  **quarto-r** to me. This is feature request already. 

* https://github.com/quarto-dev/quarto-r/issues/87 

Just need to spend some time on a new version there. 

Though this could be easy to do in **usethis** if you want something like this (as you have other project-creation function already) - everything would be done by [`quarto create project`](https://quarto.org/docs/projects/quarto-projects.html#creating-projects). 

So for the obvious place, it is also you choice. However, I would say that even if you do it in **usethis** we would do in **quarto** R package anyway.

**Comment by hadley** [comment]
I think the way forward would be for `use_quarto_project()` to be a thin wrapper over either the cli or the quarto package.

**Comment by jennybc** [comment]
How do we feel about this issue in light of the fact that `quarto::quarto_create_project()` now exists?

https://quarto-dev.github.io/quarto-r/reference/quarto_create_project.html

**Comment by hadley** [comment]
I think it's probably worth making `use_quarto_project()` (or maybe `use_project_quarto()`) that calls `quarto::quarto_create_project()`.

**Comment by jennybc** [comment]
In terms of naming and patterning off an existing thing, `create_project()` would arguably be the front-runner.

**Comment by hadley** [comment]
Yes, of course, not sure why I wrote `use` 😞
